src/main/scala/org/clulab/habitus/utils/DefaultContextExtractor.scala

```diff
@@ -87,14 +97,14 @@ class DefaultContextExtractor extends ContextExtractor {
           require(labels.length == labels.distinct.length, "Labels should be distinct in order to use them in keys of a  map.")
           val menArgLabels = arguments.map(men => men.label -> men.text).toMap
           val location   = getIrrespectiveContext(m, "Location",   thisSentLocs,   None)
+          val country = DefaultContextExtractor.regionMap.getOrElse(location, "N/A")
```

Comment:
I don't know if it is feasible or excessive, but I would favor keeping track of these Optional values a little longer as such (i.e., Option[String]) and only turn them into "N/A" (or "" for tsv/csv or null for JSON) until the very last minute.  I would find it difficult to know whether to check for "N/A" unless maybe the variable was called countryOrNA.  There are lots of N/As floating around, so I wouldn't necessary change this one, but rather review them all (at a later date, optionally).

Reply:
So, the case class DefaultContext would not have all the strings, it would have the actual types appropriate for the content of each field, and those would be converted to strings within printers?

Reply:
This printer might have a policy of converting None to "N/A".  I think you were using some kind of reflection, so this idea might not play nicely with that.  It's much more important to get the linguistics right than these little things, though.